README.md

```diff
@@ -22,17 +22,15 @@ This app illustrates the following functionality amongst other components:
 * use of UI components and extensions
 * use of functions
 
-## Basic information
-
-### Dependencies
+## Prerequisites
 
 * Foundry CLI (instructions below)
```

Comment:
Both @rlucha-crowdstrike and I started working at CrowdStrike recently. We both stumbled upon this sample and struggled because we couldn't find instructions for installing the Foundry CLI. That's the reason I added instructions for installing the CLI. The Foundry Quickstart does have these same instructions, but I feel it's useful to make them public as well. Personally, I found the Quickstart about a week after I found this sample.

Reply:
OK that's fair! This might all be easier once the main docs are publicly available.

Reply:
FWIW, we did already merge the Foundry CLI instructions in https://github.com/CrowdStrike/foundry-sample-scalable-rtr/pull/25. I didn't know there was an effort to make the main docs public. That would solve a lot of problems, IMHO! Do you know what the schedule is for this effort?

Reply:
unfortunately it's a long ways off... like multiple quarters :-(